bluesky/protocols.py

```diff
@@ -192,18 +206,20 @@ def unstage(self) -> List[Any]:
 
 @runtime_checkable
 class Pausable(Protocol):
-    def pause(self) -> Union[NoReturn, None]:
+    # TODO: Should these be Status instead?
```

Comment:
We could make this return an `Optional[Status]` I guess? Then maybe `trigger()` should do the same?

Reply:
I'd be a little bit wary of that. While it is useful to return `Optional` and `SyncOrAsync` for backwards-compatibility, do we risk making our interface too ambiguous if we do it too often? What are other people's thoughts?

Reply:
On reflection, I think anything that does I/O should return `Status` (or `Optional[Status]` for backwards compatibility) if it doesn't return a value, and `SyncOrAsync` if it does. That would make `pause` and `resume` return `Optional[Status]` and I would suggest `trigger` return `Optional[Status]` to make it consistent. This means that if inheriting from the `Readable` protocol, `trigger` doesn't need defining for `Signal`s